math/src/main/java/coppercore/math/GearConversionFunctions.java

```diff
@@ -0,0 +1,47 @@
+package coppercore.math;
+
+import static edu.wpi.first.units.Units.*;
+
+public class GearConversionFunctions {
+    public static double pitchDiameterFrom3mmPulley(int teeth) {
```

Comment:
These functions likely will fail to compile since we are returning a `Distance` but saying we will return a `double`. We should mark this and other functions to return a `Distance`